onnx/defs/parser.cc

```diff
@@ -729,19 +729,19 @@ Status OnnxParser::Parse(NodeProto& node) {
   MATCH('=');
   std::string domain("");
   std::string id;
-  ParseIdentifier(id);
+  (void)(ParseIdentifier(id));
```

Comment:
Could you comment on this? Why is this needed?

Reply:
Yes, I think this should also be a CHECK_PARSER_STATUS: I don't think an empty identifier (which is the only possible error here) is allowed here.

Reply:
One test specifies an empty id. This change is to make it green.

Reply:
Ok, thanks! I would say that the test-case is buggy, for an omitted domain, the "." is omitted too: we are supposed to use `MatMul` or `ai.onnx.MatMul` or `ai.onnx.ml.SVMClassifier`, not `.MatMul` ... but that can be fixed in a separate PR, this looks good to me now.

Reply:
@gramalingam I'm not familiar with ONNX domain format. Therefore, need your PR help if further fixes are required.